### ISSUE TITLE
cordova-plugin-googleplus fails

### ISSUE THREAD

**Issue by rasmusustav** [eclipsesource/tabris-js#1050]
When adding a plugin https://github.com/EddyVerbruggen/cordova-plugin-googleplus, the login dialog does not show up and error message writes "can only use lower 16 bits for requestCode". Is there some way I can use this plugin with tabris-js

**Comment by cookieguru** [comment]
Is this a runtime error or a build error?  If the latter, can you post the full build log?

**Comment by rasmusustav** [comment]
Its a runtime error... app builds just fine, it seems to load the plugin but instead of prompting for accounts it goes straight to error callback.

**Comment by jonek** [comment]
@rasmusustav could you provide some more context of the error message?

**Comment by mpost** [comment]
@rasmusustav We had a problem with plugins that call `cordova.getActivity().startActivityForResult(..)`. This has been fixed since the 1.10 release. Could you please try to build your app against nightly?

**Comment by rasmusustav** [comment]
@mpost could you please tell me how to do it on a local build. Sorry, I did not mention that before.

**Comment by mpost** [comment]
@rasmusustav you will have to download a nightly platform eg: https://tabrisjs.com/downloads/nightly/platforms/tabris-android.zip and than follow the regular local build procedure https://tabrisjs.com/documentation/latest/local-build#local-build

**Comment by rasmusustav** [comment]
Thanks @mpost, but I give up, the problem remains & unfortunately can't tell you anything else  - all seems to run okay, console does not show any errors except it just goes to googleplus login error callback

**Comment by cookieguru** [comment]
@rasmusustav Are there any errors in logcat?  You can view it [over adb](https://developer.android.com/studio/command-line/logcat.html), [through an app](https://play.google.com/store/search?q=logcat&c=apps&hl=en), or in [Android Studio](https://developer.android.com/studio/debug/am-logcat.html).

**Comment by rasmusustav** [comment]
@cookieguru yes there are, though I believe it's not very useful either.

11-29 01:07:29.473: I/GooglePlugin(12676): Building Google options
11-29 01:07:29.473: I/GooglePlugin(12676): Building GoogleApiClient
11-29 01:07:29.474: I/GooglePlugin(12676): GoogleApiClient built
11-29 01:07:29.474: I/GooglePlugin(12676): Trying to Log in!
11-29 01:07:29.474: E/PluginManager(12676): Uncaught exception from plugin
11-29 01:07:29.474: E/PluginManager(12676): java.lang.IllegalArgumentException: Can only use lower 16 bits for requestCode

So basically this https://github.com/EddyVerbruggen/cordova-plugin-googleplus/blob/766570abc2c40108d28849d3693d3e95d289d38f/src/android/GooglePlus.java#L173-L176 is where the problem starts and this is what @mpost mentioned should be fixed, but it just did not work for me.

**Comment by jonek** [comment]
@rasmusustav could you set up a simple demo project that can reproduce this error with the build service provided on http://tabrisjs.com ? That would have the advantage of being reproducible and we can more easily switch between different Tabris.js version and build settings there. The build service also provides detailed logging output of everything that happens during the build.

**Comment by rasmusustav** [comment]
@jonek So I did what you told - built an app with your service, used Tabris.js Client Version Nightly, got positive confirmation:

>> Put '583d7b4d634fb0e42c18c06d/io.cordova.hellocordova-debug.apk'
>> Put 1 files
Done, without errors.

And again when starting login procedure, it failed. https://github.com/rasmusustav/tabris-login-test.git

**Comment by mpost** [comment]
@rasmusustav We have solved the issue by an update to the Android cordova platform. Could you please try again with the latest nightly? 

https://tabrisjs.com/downloads/nightly/platforms/tabris-android.zip

**Comment by rasmusustav** [comment]
Oh, guys, you are the best. Thank you, the last nightly did the trick

**Comment by mpost** [comment]
@rasmusustav Glad to hear that. :)
